astrocut/tests/test_cutout_processing.py

```diff
@@ -247,14 +247,18 @@ def test_center_on_path(tmpdir):
     assert primary_header["OBJECT"] == "Test Target"
 
     # Using the default output filename and not giving an image wcs
```

Comment:
I believe this test case was accidentally removed. Can be one of the cases in parametrize.

Reply:
This first test looks fine so I'm going to leave it because if I moved it to another function it would just take more lines to rewrite getting the ffis and the cutout_maker etc. 

The original tests for `test_center_on_path` was:
1. Testing that the output file is named `mt_cutout.fits` even tho you're given an input target (I believe)
2. Testing that the output file is named `path_[...].fits` when target is set to none, like you mention below. 


So I'm going to leave 1 as is and parametrize the target input for 2: where target = None and then target = "C\2104 [....]"